Makefile

```diff
@@ -0,0 +1,65 @@
+#============================  LIBS & INCLUDES
+
+EIGEN_PATH = ./pyBeam/externals/Eigen
+CODI_PATH = ./pyBeam/externals/CoDiPack/include
+
+INCLPATH =  -I$(EIGEN_PATH) -I$(CODI_PATH) $(PYBEAM_INCLUDEPY)
```

Comment:
Where is `PYBEAM_INCLUDEPY` defined?

Reply:
You define it locally in your computer, by doing:
`export PYBEAM_INCLUDEPY="$(python-config --includes)"`
or adding that to your .bashrc. That takes your local python config and defines where the include file is for the compilation.